demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1807,103 +1773,104 @@ static OtaHttpStatus_t httpInit( char * pUrl )
-    HTTPRequestInfo_t requestInfo;
+    HTTPRequestInfo_t xRequestInfo;
     /* Represents a response returned from an HTTP server. */
-    HTTPResponse_t response;
+    HTTPResponse_t xResponse;
     /* Represents header data that will be sent in an HTTP request. */
-    HTTPRequestHeaders_t requestHeaders;
+    HTTPRequestHeaders_t xRequestHeaders;
 
     /* Return value of all methods from the HTTP Client library API. */
-    HTTPStatus_t httpStatus = HTTPSuccess;
+    HTTPStatus_t xHTTPStatus = HTTPSuccess;
 
     /* Reconnection required flag. */
     bool reconnectRequired = false;
 
     /* Initialize all HTTP Client library API structs to 0. */
-    ( void ) memset( &requestInfo, 0, sizeof( requestInfo ) );
-    ( void ) memset( &response, 0, sizeof( response ) );
-    ( void ) memset( &requestHeaders, 0, sizeof( requestHeaders ) );
+    ( void ) memset( &xRequestInfo, 0, sizeof( xRequestInfo ) );
+    ( void ) memset( &xResponse, 0, sizeof( xResponse ) );
+    ( void ) memset( &xRequestHeaders, 0, sizeof( xRequestHeaders ) );
 
     /* Initialize the request object. */
-    requestInfo.pHost = serverHost;
-    requestInfo.hostLen = serverHostLength;
-    requestInfo.pMethod = HTTP_METHOD_GET;
-    requestInfo.methodLen = sizeof( HTTP_METHOD_GET ) - 1;
-    requestInfo.pPath = pPath;
-    requestInfo.pathLen = strlen( pPath );
+    xRequestInfo.pHost = serverHost;
+    xRequestInfo.hostLen = serverHostLength;
+    xRequestInfo.pMethod = HTTP_METHOD_GET;
+    xRequestInfo.methodLen = sizeof( HTTP_METHOD_GET ) - 1;
+    xRequestInfo.pPath = pPath;
+    xRequestInfo.pathLen = strlen( pPath );
 
     /* Set "Connection" HTTP header to "keep-alive" so that multiple requests
      * can be sent over the same established TCP connection. */
-    requestInfo.reqFlags = HTTP_REQUEST_KEEP_ALIVE_FLAG;
+    xRequestInfo.reqFlags = HTTP_REQUEST_KEEP_ALIVE_FLAG;
 
     /* Set the buffer used for storing request headers. */
-    requestHeaders.pBuffer = httpUserBuffer;
-    requestHeaders.bufferLen = HTTP_USER_BUFFER_LENGTH;
+    xRequestHeaders.pBuffer = httpUserBuffer;
+    xRequestHeaders.bufferLen = HTTP_USER_BUFFER_LENGTH;
 
-    httpStatus = HTTPClient_InitializeRequestHeaders( &requestHeaders,
-                                                      &requestInfo );
+    xHTTPStatus = HTTPClient_InitializeRequestHeaders( &xRequestHeaders,
+                                                       &xRequestInfo );
 
-    HTTPClient_AddRangeHeader( &requestHeaders, rangeStart, rangeEnd );
+    HTTPClient_AddRangeHeader( &xRequestHeaders, ulRangeStart, ulRangeEnd );
 
-    if( httpStatus == HTTPSuccess )
+    if( xHTTPStatus == HTTPSuccess )
     {
         /* Initialize the response object. The same buffer used for storing
          * request headers is reused here. */
-        response.pBuffer = httpUserBuffer;
-        response.bufferLen = HTTP_USER_BUFFER_LENGTH;
+        xResponse.pBuffer = httpUserBuffer;
```

Comment:
httpUserBuffer -> pucHttpUserBuffer

Reply:
Done in https://github.com/aws/amazon-freertos/pull/3148/commits/6bbe8a3c217e31fff82104a685ad811120104724